www/on/%platform/%user_name/failure.html.spt

```diff
@@ -29,20 +24,25 @@ title = _("Failure")
 
     {% if team %}
 
-    <h2>Are you really an administrator of the {{ user_name }} team?</h2>
+    <h2>{{ _("Are you really an administrator of the {0} team?", user_name) }}</h2>
 
-    <p>Your attempt to {{ action }} this account failed because you're
```

Comment:
Unfortunate that we lost the distinction between different actions here. I just noticed at https://github.com/gratipay/gratipay.com/commit/5f3fe6eb978a0b531f420e32f9848000103b2a47#commitcomment-10237459.